nucypher/blockchain/eth/interfaces.py

```diff
@@ -75,9 +77,17 @@ class BlockchainInterface:
     ethereum contracts with the given web3 provider backend.
     """
 
-    TIMEOUT = 180  # seconds
+    TIMEOUT = 600  # seconds
     NULL_ADDRESS = '0x' + '0' * 40
 
+    DEFAULT_GAS_STRATEGY = 'medium'
+    GAS_STRATEGIES = {'glacial': time_based.glacial_gas_price_strategy,
+                      'slow': time_based.slow_gas_price_strategy,
+                      'medium': time_based.medium_gas_price_strategy,
+                      'fast': time_based.fast_gas_price_strategy,
+                      # 'rpc': gas_strategies.rpc.rpc_gas_price_strategy  # TODO: Investigate compatibility with rpc
```

Comment:
May be better to have an issue for that TODO

Reply:
:heavy_check_mark: - Removed for now.